test/test-values/multiple_apps_with_single_container.yaml

```diff
@@ -0,0 +1,17 @@
+apps:
+  example:
+    enabled: true
+    type: deployment
+    containers:
+      example:
+        image:
+          repository: nginx
+          tag: latest
+  example2:
+    enabled:  true
+    type: deployment
+    containers:
+      example:
+        image:
+          repository: nginx
+          tag: 9.9.9
```

Comment:
a few new lines hanging around you might want to clean up

Reply:
I'm not quite sure what you mean here? the files have no trailing whitespace, do you mean the lack of newline at the end of the file?